src/lib/evil/unposix/strings.c

```diff
@@ -0,0 +1,10 @@
+#include "strings.h"
+
+inline int
```

Comment:
No `inline` in this case. It just makes sense if it were defined in a header.

Reply:
i'm confused, it is defined into strings.h